backend/user_types.py

```diff
@@ -0,0 +1,13 @@
+from pydantic import BaseModel
+from typing import Optional, List
+
+
+class UserDetails(BaseModel):
+    email: str
```

Comment:
Username is not better then email? You can put an Email when registering

Reply:
not exactly. I thought about it for a while and i initally wanted the user to put also a username, but i think its pretty irrelevant for us. working with mails is fine in my opinion. later on we can ask for the full name which can be much better data to render